templates/citizen_action/_layout.html.twig

```diff
@@ -6,6 +6,30 @@
 
 {% block javascripts %}
     {{ parent() }}
+    <script type="text/javascript">
+        Kernel.onLoad(function(di) {
+            {% if is_granted('ROLE_ADHERENT') %}
```

Comment:
We should add a new function like `can_unregister_citizen_action` and use it here